flow/project.py

```diff
@@ -727,17 +736,25 @@ def __str__(self):
         return f"{type(self).__name__}(op_func='{self._op_func}')"
 
     def __call__(self, *jobs):
+        r"""Call the operation on the provided jobs.
+
+        :param \*jobs:
+            The jobs passed to the operation.
+        :type \*jobs:
+            One or more instances of :class:`.Job`.
+        """
         return self._op_func(*jobs)
 
 
 class FlowGroupEntry:
     """A FlowGroupEntry registers operations for inclusion into a :py:class:`FlowGroup`.
```

Comment:
```suggestion
    """A FlowGroupEntry registers operations for inclusion into a :class:`FlowGroup`.
```

Reply:
I also stripped `:py:` prefixes in a later PR.